docs/csharp/language-reference/compiler-messages/cs0051.md

```diff
@@ -12,17 +12,26 @@ ms.assetid: 62182e8d-c4a5-4853-a990-fd57a4f7c3b8
 
 Inconsistent accessibility: parameter type 'type' is less accessible than method 'method'
 
- The return type and each of the types referenced in the formal parameter list of a method must be at least as accessible as the method itself. Make sure the types used in method signatures are not accidentally private due to the omission of the `public` modifier. For more information, see [Access Modifiers](../../programming-guide/classes-and-structs/access-modifiers.md).
+This error occurs when you declare a method (including constructors) with a parameter type that is less accessible than the method itself. For example, you might have a public constructor that uses an internal or private class as a parameter type.
 
-## Example
+The most common scenario is when you're defining a public constructor but one of its parameter types is internal or private. This creates an inconsistency because external code can see the constructor but can't access the types needed to call it.
```

Comment:
It's not necessarily constructors only:

```suggestion
The most common scenario is when you're defining a public method but one of its parameter types is internal or private. This creates an inconsistency because external code can see the constructor but can't access the types needed to call it.
```